src/couch_util_clock.erl

```diff
@@ -0,0 +1,217 @@
+%% Copyright (c) 2011-2014, Loïc Hoguin <essen@ninenines.eu>
```

Comment:
Since it's not our code, update of NOTICE file in apache/couchdb repository is required.